apps/ejabberd/src/jlib.erl

```diff
@@ -54,7 +54,6 @@
          parse_xdata_submit/1,
          timestamp_to_iso/1, % TODO: Remove once XEP-0091 is Obsolete
```

Comment:
Just noticed that there is another obsolete function. I'd remove it also and see if our tests run by travis passes.